source/operator.rst

```diff
@@ -514,3 +514,320 @@ options for Percona Monitoring and Management.
 |                 | The PMM Server password should be configured using Secrets                                |
 +-----------------+-------------------------------------------------------------------------------------------+
 
+.. _operator.backup-section:
+
+`Backup Section <operator.html#operator-backup-section>`_
+--------------------------------------------------------------------------------
+
+The ``backup`` section in the
+`deploy/cr.yaml <https://github.com/percona/percona-server-mysql-operator/blob/main/deploy/cr.yaml>`__
+file contains the following configuration options for the regular
+Percona XtraDB Cluster backups.
+
+.. tabularcolumns:: |p{2cm}|p{13.6cm}|
+
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _backup-enabled:                                                                       |
+|                 |                                                                                           |
+| **Key**         | `backup.enabled <operator.html#backup-enabled>`_                                          |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value Type**  | boolean                                                                                   |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``true``                                                                                  |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Description** | Enables or disables making backups                                                        |
++-----------------+-------------------------------------------------------------------------------------------+
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _backup-image:                                                                         |
+|                 |                                                                                           |
+| **Key**         | `backup.image <operator.html#backup-image>`_                                              |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value**       | string                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``percona/percona-xtrabackup:{{{pxbrecommended}}}``                                             |
```

Comment:
In this release it will be `percona/percona-server-mysql-operator:0.2.0-backup`, so we will not use official PXB image.